src/app.module.ts

```diff
@@ -1,9 +1,10 @@
 import { Module } from '@nestjs/common';
```

Comment:
Name All Module In plural not author naem it authors rename every class and every file out ouf dtos and interfaces